tss-esapi/src/abstraction/nv.rs

```diff
@@ -63,6 +65,8 @@ fn get_nv_index_info(
 }
 
 /// Lists all the currently defined NV Indexes' names and public components
+///
+/// NOTE: This call _may_ close existing ESYS handles to the existing NV Indexes.
```

Comment:
Why do you say "_may_"?

Reply:
Because I assume on "old" versions of tpm2-tss, it doesn't, and that's why I have to do `unwrap_or_else` - that handle is still open, since the TSS (probably) created new ESYS handles every time.